test_suite/tests/expandtest.rs

```diff
@@ -0,0 +1,7 @@
+#[cfg(not(target_os = "emscripten"))]
+#[rustversion::attr(not(nightly), ignore)]
```

Comment:
Would it be possible to also #\[ignore\] if cargo-expand is not installed? We can add a build.rs to the test_suite crate to detect cargo-expand.

Reply:
Yes, I'll try.

Reply:
Sounds good. Later it could be its own crate: https://github.com/dtolnay/request-for-implementation/issues/44

Reply:
Done.